nav2_bringup/params/nav2_params.yaml

```diff
@@ -115,6 +116,16 @@ controller_server:
       plugin: "nav2_controller::SimpleGoalChecker"
       xy_goal_tolerance: 0.25
       yaw_goal_tolerance: 0.25
+      path_length_tolerance: 1.0
+    PathHandler:
```

Comment:
@mini-1235 isn't `max_robot_pose_search_dist` missing?

Reply:
No strong opinion, I just thought that we expose in the yaml file everything that is a configurable parameter. But if the philosophy is rather to expose parameters that are likely to be tuned then I also understand that